# Rang value
# Write joint value read from QR code.
rang_value:
  - 0
  - 0
  - 0
  - 0
  - 0
  - 0